src/worker/controller/actions/sync/IdentifyMissingStatesAction.js

```diff
@@ -43,6 +43,10 @@ class IdentifyMissingStatesAction {
             }
             return finalCallback(error);
           }
+          let localTips = tips;
```

Comment:
Why not just do?
```
if (!tips) {
	tips = []
}
```

And I think that if `tips` in undefined then `let localTips = tips;` will also fail, but i'm not sure.

Reply:
As a practice of not changing the received parameters (that might be passes by a reference).. Updated the code

Reply:
Didn't know that. good to know